usagov/vote-gov-drupal
# VOTE-1568 Resolve issue with absolute urls for css and other assets

## Jira ticket (required)
[VOTE-1568](https://bixal-projects.atlassian.net/browse/VOTE-1568) - Resolve issue with absolute urls for css and other assets
[VOTE-1369](https://bixal-projects.atlassian.net/browse/VOTE-1369) - Static Site fonts are broken

## Description (optional)
Insert a brief summary of the changes included in this PR and any additional information or context which may help the reviewer.

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. Insert step one

### Post-deploy steps
1. Trigger the Build Static Workflow to generate the static site via Tome (CircleCI).

### Testing steps
1. Visit the CMS site and confirm using Developer Tools that there are no css or asset references which are not found or are resulting in CORS issues.
2. Review the metadata and confirm that there aren't any images following the pattern {domain}//{asset_url}.
3. Repeat steps 1 & 2 for the Static site.
